### DISCUSSION TITLE
Tabs 组件初始化时，其他tab内容的dom树没有初始化值

### DISCUSSION THREAD

**Discussion by yangmeiz** [youzan/vant#12357] [General]
Tabs 组件嵌套form表单，form表单有必填项，点击保存时无法校验其他tab中的必填项（Tabs 组件初始化时，其他tabs中的数据没有加载dom树中）

**Comment by inottn** [comment]
lazy-render 设置为 false

**Reply by yangmeiz** [reply]
嗯呢，可以啦，谢谢
